stardis/opacities/base.py

```diff
@@ -61,88 +56,78 @@ def calc_tau_h_minus(
 
     Returns
     -------
-    tau_h_minus : numpy.ndarray
+    alpha_h_minus : numpy.ndarray
         Array of shape (no_of_shells, no_of_frequencies). H minus optical
         depth in each shell for each frequency in tracing_nus.
     """
 
+    fv_geometry = stellar_model.fv_geometry
+
     wbr_cross_section = read_wbr_cross_section(wbr_fpath)
 
     tracing_lambdas = tracing_nus.to(u.AA, u.spectral()).value
 
     # sigma
     h_minus_sigma_nu = np.interp(
-        tracing_lambdas,
-        wbr_cross_section.wavelength,
-        wbr_cross_section.cross_section,
+        tracing_lambdas, wbr_cross_section.wavelength, wbr_cross_section.cross_section,
     )
 
-    # tau = sigma * n * l; shape: (num cells, num tracing nus) - tau for each frequency in each cell
-    tau_h_minus = (
-        h_minus_sigma_nu
-        * (np.array(splasma.h_minus_density) * marcs_model_fv.cell_length.values)[
-            None
-        ].T
-    )
-    return tau_h_minus
+    # alpha = sigma * n * l; shape: (num cells, num tracing nus) - alpha for each frequency in each cell
```

Comment:
Yes, that's correct. I will see if I can fix that without messing up all of the rebasing, otherwise I can fix that in my next PR (which it will happen to be relevant to).